src/index.html

```diff
@@ -1,13 +1,370 @@
+<!-- #region head -->
 <!DOCTYPE html>
-<html lang="en">
+<html lang="en" class="page">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Miami</title>
+    <title>THE MET</title>
+    <link
+      rel="shortcut icon"
+      href="./images/met-logo.png"
+      type="image/x-icon"
+    >
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Cinzel:wght@500;600;700&display=swap" rel="stylesheet">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto&display=swap" rel="stylesheet">
     <script src="scripts/main.js" defer></script>
   </head>
-  <body>
-    <h1>Landing Page</h1>
+  <!-- #endregion -->
+  <body class="page__body">
+    <header class="page__header header">
+      <div class="header__top-bar top-bar">
+        <a href="#" class="top-bar__logo">
+          <img
+            src="images/met-logo.svg"
+            alt="THE MET logo"
+            class="top-bar__logo__image"
+          >
+        </a>
+
+        <div class="top-bar__icons">
+          <a href="#phone" class="icon icon--phone"></a>
```

Comment:
I cannot click on numbers because if I move on cursor away from icon `#phone` disappear. Think how we can fix this behaviour.